numba/npyufunc/array_exprs.py

```diff
@@ -0,0 +1,349 @@
+from __future__ import print_function, division, absolute_import
```

Comment:
I wonder if it would make sense a directory for rewrites. I am not sure if I would expect a rewrite in a file named "array_exprs.py" inside npyufunc. It may be early though, but if we start adding rewrites it would make sense having a place for them.